<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>780</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CLOSE</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>831</width>
      <height>771</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="scan_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
    <widget class="QComboBox" name="serial_comboBox">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>10</y>
       <width>231</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="speed_comboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="open_btn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>610</y>
       <width>811</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="start_serial_btn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_serial_btn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recording_btn">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>REC</string>
     </property>
    </widget>
    <widget class="PlotWidget" name="rtd1_plot" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>110</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="PlotWidget" name="rtd3_plot" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>370</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>90</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RTD1 (째C)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>350</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RTD3 (째C)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="recording_bar">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>45</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="PlotWidget" name="rtd2_plot" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>240</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>220</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RTD2 (째C)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="stoprecording_btn">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>STOPREC</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>480</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RTD4 (째C)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="PlotWidget" name="rtd4_plot" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>500</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="manual_btn">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>15</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>M</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="data_len_input">
     <property name="geometry">
      <rect>
       <x>495</x>
       <y>15</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="auto_btn">
     <property name="geometry">
      <rect>
       <x>455</x>
       <y>15</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>87</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>HEATER #1 --&gt; RTD</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="PlotWidget" name="heater1_pwm_plt" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="PlotWidget" name="heater2_pwm_plt" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="PlotWidget" name="heater3_pwm_plt" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="PlotWidget" name="heater4_pwm_plt" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>500</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="heater1_rtd_sel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>217</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>HEATER #2 --&gt; RTD</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="heater2_rtd_sel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>347</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>HEATER #3 --&gt; RTD</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="heater3_rtd_sel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>340</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>477</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>HEATER #4 --&gt; RTD</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="heater4_rtd_sel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>470</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>3</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="heater1_temp_set">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>83</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="heater2_temp_set">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>213</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="heater3_temp_set">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>343</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="heater4_temp_set">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>473</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>85</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>째C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>215</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>째C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>345</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>째C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>475</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>째C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="set_heater_btn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>470</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
